content/en/docs/started/workstation/kubeflow-on-microk8s.md

```diff
@@ -7,7 +7,9 @@ weight = 60
 
 ## Introduction
 
-This guide describes how to deploy and run Kubeflow using [MicroK8s](https://microk8s.io/) - a small enterprise Kubernetes cluster — locally on built-in hypervisors.
+This guide describes how to deploy and run Kubeflow on [MicroK8s](https://microk8s.io/) - a small enterprise Kubernetes cluster — locally on built-in hypervisors.
```

Comment:
I think this isn't very clearly expressed. Perhaps something like:
This guide describes how to deploy and run Kubeflow locally with [MicroK8s](https://microk8s.io/) - a small enterprise Kubernetes cluster.

Reply:
Thank you for the input @evilnick! I'll update